### PR TITLE
APR-2985-Gateway-checks-for-prefixed-charities-bug-fix

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

**Review by MarkFCain — CHANGES_REQUESTED** [review]
this is an oc

File: src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs [hunk 1]
```diff
@@ -110,7 +110,7 @@ private async Task LookupCharityCommissionDetails(ApplyGatewayDetails applyGatew
                 var message =
                     $"Unable to parse charity registration number from charity number in ukrlp: '{charityNumberFromUkrlp}'";
                 _logger.LogError(message);
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs:112]: this is no longer an error, but an occasional, but expected condition.  Can you change .LogError to .LogInformation

Comment (DevenKShah) [reply src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs:112]: Perhaps change the message as well. To something like `Charity registration number cannot be parsed into a number, skipping getting details from charity commission for {charityNumber}`

**Review by MarkFCain — DISMISSED** [review]

File: src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs (hunk 1 above)
Comment (raviudari01) [reply src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs]: Done

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**